smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -410,6 +441,42 @@ def infra_ddict(self) -> str:
 
         return str(self._infra_ddict.serialize())
 
+    @staticmethod
+    def create_run_policy(
+        request: DragonRequest, node_name: str
```

Comment:
Can we add the proper doc strings to this method?

Reply:
great catch! added